src/Bridges/ApplicationLatte/TemplateFactory.php

```diff
@@ -112,4 +115,12 @@ public function createTemplate(UI\Control $control = NULL)
 		return $template;
 	}
 
+	public function setTemplateClass($templateClass)
+	{
+		if (!class_exists($templateClass) || !is_a($templateClass, UI\ITemplate::class, TRUE)) {
```

Comment:
Why not `ITemplate`? Can you please elaborate it more @dg? I don't fully understand... :) Thank you.